integration-test/src/features/organizations.feature

```diff
@@ -0,0 +1,43 @@
+Feature: All about Organizations
+
+  Background:
+    Reporting Enrollment running
+
+  Scenario: An organization creates an enrollment
+    Given a random id organization
+    When the organization "creates" enrollment
```

Comment:
i'm not sure but `"` are needed for string ?

Reply:
Yes, matches single-quoted or double-quoted strings, ref: https://github.com/cucumber/cucumber-expressions#readme